### ISSUE TITLE
Update deployment methods to switch away from TimescaleDB

### ISSUE THREAD

**Issue by caugustus-sourcegraph** [sourcegraph/sourcegraph#32075]
Code Insights has begun deprecating TimescaleDB, and will switch to a vanilla Postgres image soon (tentatively scheduled for 3.38).

The vanilla Postgres image [runs as a different uid](https://github.com/sourcegraph/sourcegraph/blob/main/docker-images/postgres-12.6-alpine/Dockerfile#L3) than Timescale, so there will be a mismatch between the image and the files on disk. The proposed solution is to add an init container / standalone fix that will `chown` the existing files to the new uid (999).

https://github.com/sourcegraph/sourcegraph/issues/24430#issuecomment-1028392916

**Comment by caugustus-sourcegraph** [comment]
On hold until post-3.38 release.

**Comment by virginiaulrich** [comment]
Currently blocked by https://github.com/sourcegraph/sourcegraph/issues/32918

**Comment by caugustus-sourcegraph** [comment]
Still blocked by #32918 (which is blocked by CI failures)
